client/blocks/importer/wordpress/content-chooser/index.tsx

```diff
@@ -81,7 +82,7 @@ export const ContentChooser: React.FunctionComponent< Props > = ( props ) => {
 					subHeaderText={ __( 'Choose what you would like to import to your new site.' ) }
 				/>
 				<div className={ 'step-wrapper__header-image' }>
-					<img alt="" src="/calypso/images/importer/onboarding-2.svg" />
+					<img alt="Import" src={ illustrationImg } />
```

Comment:
Remember translations `__()`

Reply:
That's true.

There is no usable information from the accessibility perspective. I'll pass through the code base and add `aria-hidden` attributes.